### PR TITLE
Support parsing multiple show variables.

### PR DESCRIPTION
**Pull request by francis-du**
Support parsing multiple show variables.

eg:

```sql
SHOW STORAGE GROUP
SHOW TIMESERIES root.ln.wf01.wt01.status
```

### CODE REVIEW

**Review by andygrove — APPROVED** [review]
LGTM

File: src/parser.rs [hunk 1]
```diff
@@ -1726,6 +1729,19 @@ impl<'a> Parser<'a> {
...
+        let mut idents = vec![];
+        loop {
+            match self.next_token() {
+                Token::Word(w) => idents.push(w.to_ident()),
+                Token::EOF => break,
+                _ => {}
+            }
+        }
+        Ok(idents)
+    }
```
Comment (edrevo) [inline src/parser.rs:1743]: Is `SHOW` (without any variable) valid SQL? This code would return  `Ok(vec![])` for the SQL code `SHOW`, correct?

Comment (francis-du) [reply src/parser.rs:1743]: yes

**Comment by francis-du** [comment]
> @francis-du could you rebase please

Sure

**Comment by Dandandan** [comment]
@francis-du I think you didn't rebase/push the rebase yet?

**Comment by francis-du** [comment]
> @francis-du I think you didn't rebase/push the rebase yet?

Done.

**Review by Dandandan — APPROVED** [review]

**Comment by Dandandan** [comment]
Thanks @francis-du 👍

**Pull request merged**